MovieMVVM/MovieMVVM/Module/DetailModule/View/Cell/ActorCollectionViewCell.swift

```diff
@@ -59,18 +53,19 @@ final class ActorCollectionViewCell: UICollectionViewCell {
 
     // MARK: - Public Metods
 
-    func setupActor(_ actror: Actor) {
-        guard let urlImage = URL(string: "\(Constants.getImageURL) \(actror.actorImageName)") else { return }
-        let session = URLSession(configuration: .default)
-        let task = session.dataTask(with: urlImage) { data, _, error in
-            guard let data = data, error == nil else { return }
-            DispatchQueue.main.async {
-                let image = UIImage(data: data)
-                self.actorImageView.image = image
+    func congigureCell(_ actror: Cast, viewModel: MovieDetailViewModelProtocol) {
+        actorNameLabel.text = actror.name
+        actorRoleLabel.text = actror.character
+        guard let path = actror.profilePath else { return }
+        viewModel.imageService?.loadImage(path: path) { result in
```

Comment:
```suggestion
        viewModel.imageService?.loadImage(path: path) { result in
```
так не пойдет это все даем в vm в ячейку иньектим vm и тут только кложуры